### PR TITLE
Group categories in country page

### PR DESCRIPTION
**Pull request by gjchong25**
Groups messages by category in the country page.

### CODE REVIEW

File: src/main/webapp/WEB-INF/category.jsp [hunk 1]
```diff
@@ -52,7 +52,7 @@
     <button onclick="getLocation()">Add your location</button>
     <div id="map"></div>
     <% }  %>
-    <div id="message-container">
+    <div class="message-container">
```
Comment (nbaldy) [inline src/main/webapp/WEB-INF/category.jsp:63]: I'm curious, what does this change do?

**Review by nbaldy — COMMENTED** [review]
This looks good to me, but can you merge it with master so that we can make sure all the functionality still works? Because now we are making so many changes that merging is super complex

File: src/main/webapp/WEB-INF/category.jsp (hunk 1 above)
Comment (gjchong25) [reply src/main/webapp/WEB-INF/category.jsp:63]: It doesn't make a difference in terms of functionality I believe. I think class is for repeated divs, and id is for one-time-only type divs? Not too sure though.

**Review by nbaldy — APPROVED** [review]
Looks good! I'm going to go ahead and merge

**Pull request merged**